client/my-sites/importer/author-mapping-item.jsx

```diff
@@ -31,22 +32,17 @@ export default class extends React.PureComponent {
 				avatar_URL: PropTypes.string.isRequired,
 			} ),
 		} ).isRequired,
+		currentUser: PropTypes.object,
 	};
 
 	componentWillMount() {
```

Comment:
We could use this opportunity to rename this method to `componentDidMount`. It will be more Fiber compatible then.

Reply:
done